WordPress/Sniffs/VIP/PluginMenuSlugSniff.php

```diff
@@ -40,7 +40,7 @@ class PluginMenuSlugSniff extends \WordPress\Sniffs\Security\PluginMenuSlugSniff
 	);
 
 	/**
-	 * Don't use.
+	 * Process the token and handle the deprecation notices.
```

Comment:
Please undo this change. This sniff was not touched by this PR. The description in this case is correct as the sniff has been moved (in a previous PR) and this version of the sniff should no longer be used.